js/driver/virtio/vring/index.js

```diff
@@ -55,6 +55,9 @@ VRing.prototype.fetchBuffers = function(fn) {
     for (;;) {
       let u8 = this.getBuffer();
       if (null === u8) {
+        if (done) {
+          done();
+        }
```

Comment:
No, I forgot to remove that. I *was* going to use it, but then I found another solution.